frontend/app/views/spree/products/index.html.erb

```diff
@@ -1,7 +1,7 @@
 <% content_for :sidebar do %>
   <div data-hook="homepage_sidebar_navigation">
     <% if "spree/products" == params[:controller] && @taxon %>
-      <%= render :partial => 'spree/shared/filters' %>
+      <%= render partial: Spree::Config.search_filter_presenter_class.partial_path %>
```

Comment:
if we instantiate the presenter class before the partial and pass it in as a parameter to the view we'd be able to make the partial_path an instance method and give stores tighter control of how things can be extended